lang/en/lang.php

```diff
@@ -58,3 +58,8 @@
 $lang['preparingdownload']      = "We are preparing your download, please wait...";
 $lang['titleerrordownload']     = "Error";
 $lang['faileddownload']         = "There was a problem generating your download.";
+
+$lang['select_namespace']       = "Add Namespace to Selection";
```

Comment:
Hmmm...both is fine for me. Choose your favourite text.

So you are going to change the clear button to an icon in the near future? Then we should maybe stay with text for now. And if you go for an icon, then go for an icon for both buttons.